oci-ignite-advanced/intro/intro.md

```diff
@@ -0,0 +1,47 @@
+# Introduction
+
+The labs in this workshop cover a level 100 set of skills for **Oracle Cloud Infrastructure (OCI) Core Services**, including Virtual Cloud Networks (VCN), Autonomous Database, Oracle Kubernetes Engine (OKE). These labs also provide some of the foundational skills and knowledge you will need to attempt the [Oracle Cloud Infrastructure Architect Associate Certification](https://education.oracle.com/oracle-cloud-infrastructure-2022-architect-associate/pexam_1Z0-1072-22) offered by Oracle University.
+
+The Oracle Cloud Infrastructure Architect Associate exam is designed for individuals who possess a strong foundational knowledge of architecting infrastructure using Oracle Cloud Infrastructure (OCI) services. The certification is intended for candidates with technical backgrounds who want to validate their understanding of Oracle Cloud Infrastructure.
+
+The labs featured on this page cover various solutions, skill levels, and categories based on Oracle Cloud Infrastructure (OCI). 
+
+Deploy a complete data warehousing platform that scales to both your technical and analytic requirements.
+
+
+Estimated Workshop Time: 1 hour and 30 minutes
+
+### Objectives
+
+In this workshop, you will:
+
+- Get introduced to the regions available on Oracle Cloud
+- Apply the Ignite Basics knowledge
+- Create a Virtual Cloud Network
+- Create an Autonomous Database
+
+**<font color="blue">Optional</font>**:
+ - Create a Kubernetes Cluster
+### Prerequisites
+* [Familiarity with OCI console](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/console.htm)
+* [Overview of Networking](https://docs.oracle.com/en-us/iaas/Content/Network/Concepts/overview.htm)
+* [Oracle Cloud basics](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/concepts.htm)
+* [Connecting to a compute instance](https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/accessinginstance.htm)
+* Your **<font color="red">Oracle Cloud Account</font>** - During this workshop, we will create a Level 100 environment for you to use on your tenancy.
+
+
+## About Oracle Cloud Infrastructure
+
+Oracle Cloud Infrastructure is a set of complementary cloud services that enable you to build and run a wide range of applications and services in a highly available hosted environment. Oracle Cloud Infrastructure offers high-performance compute capabilities (as physical hardware instances) and storage capacity in a flexible overlay virtual network that is securely accessible from your on-premises network.
+
+### Let's get Started!
```

Comment:
Remove the Let's get started section and add You may now **proceed to the next lab**.